### PR TITLE
Emergency Response Team

### PR DESCRIPTION
**Pull request by Peptide90**
## About the PR 
**REQUIRES #8512 and #8301 **

Revival of the PR from January. The required components I needed are in / nearly in so thought I should bring it back as a MVP of ERT.

**TO DO:**
- [x] Tweak access for the ERT / Centcom card, possibly make them a unique shuttle computer and ID lock it. Make the ID level not show on HoPs computer.
- [x] Add Exosuit slot item to the EVA loadout for oxygen tank.
- [x] Buff ERT hardsuits so they're not slow.
- [x] Change hardsuits to incorporate the helmet
- [x] Rework ERT loadout for weapons - Someone familiar with weapons can have input into lore friendly stuff.
- [x] ERT Backpack fills
- [x] Map ERT shuttle with spawns

**FOR A FUTURE PR**
- Implement ERT into comms console or a card swipe mechanic.
- Interface ERT calls with Centcom somehow so they can be rejected and so centcom can change the makeup of the team (or make it random).
- Figure out the spawning of the ERT shuttle. This can be done via a unique ERT shuttle with spawners and it will appear on the main menu as a station people join into, otherwise it needs to be like how nukies spawn and make it a ghost role.

**Screenshots**

ReBbrnjllD
ERT 1
This gif is pre icons. Icons are shown below in screenshot. Also loadouts in gif not final.
lh4rkHbihU
ENTITY MENU
**Changelog**

:cl:
- add: The Emergency Response Team have arrived! Look out for ghost roles becoming available when centcom advises they're coming. (Not currently call-able, ask an admin via communications console to centcom).

### CODE REVIEW

**Comment by Pangogie** [comment]
[6697] Presto name-o.

File: Resources/Prototypes/Entities/Mobs/Player/human.yml [hunk 1]
```diff
@@ -48,3 +48,181 @@
...
+    sprite: Markers/jobs.rsi
+    state: ertleader
+  - type: GhostRoleMobSpawner
+    prototype: MobHumanERTLeader
+    deleteOnSpawn: true
+    makeSentient: false
+    name: ERT Leader
+    description: Lead a team of specialists to resolve the stations issues.
+  - type: UtilityAI
+    startingGear: ERTLeaderGear
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/Player/human.yml:68]: Just make a new component or something at least please.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Mobs/Player/human.yml (hunk 1 above)
Comment (Peptide90) [reply Resources/Prototypes/Entities/Mobs/Player/human.yml:68]: These are takeovers yes for admins to spawn. There are spawners too for use with mapped shuttles and for events firing in future.

The UtilityAI doesn't allow movement on takeover hence the tick box issue, making a component for it is probably (very much likely) out of my ability. So at least this is just a big content PR.

**Comment by Peptide90** [comment]
> #6697  Presto name-o.

I do wish. Mirror suggested just making the name system that uses the datasets but probs above my head and definitely above my time available to do it competently.

**Comment by metalgearsloth** [comment]
So normally I'm okay with placeholder code as long as it's not decreasing the overall code quality, but to my knowledge I don't think anything else in the codebase has the dummy UtilityAI so I'd prefer that was fixed before merging.

Gonna derro this until someone has the time to do the 2 issues first.

**Comment by Peptide90** [comment]
Reopening as multiple steps closer. Latest commit requires #8512 for loadouts to not cause a failed build.

Will need to look over the loadouts again before this is ready to merge to make sure they have the correct weapons and equipment.

This PR is also purely admeme until someone adds the swipe card thing or implements it into the console. Ideally centcom should have some granular control on rejecting the requests and changing the makeup of the ERT. Someone with C# knowledge can handle the spawning of it but that can all be a TODO.

**Comment by EmoGarbage404** [comment]
Will the ERT spawners be placed on the shuttle map? I think that would be the best way to implement it until a more robust ghost role system is implemented.

**Comment by Peptide90** [comment]
> 

This was my thinking, I can create a map prototype for the ERT shuttle then when the shuttle is spawned it will open it up as another "station" that can be joined from the main screen. This allows us to have a set ERT makeup but won't be robust in scaling it or deciding e.g. just a janitor ERT.

I'll polish this up and if anyone wants to get more advanced with game rules / spawning / calling they can. Because I made all the ERT as entities it's a 1 click job to spawn more as ghost roles as they spawn fully configured as ghost roles, randomised and kitted out, so admins can increase the ERT presence if needed.

**Comment by Peptide90** [comment]
Should be good to go now. Shuttle is mapped with 5 spawners on, hopefully this means when someone runs loadgamemap ShuttleERT, it opens it up as a station on the lobby with the 5 job slots open.

Currently admin spawn only because making a 2 card swipe system to spawn it is above me.

**Pull request merged**